src/main/java/com/answerdigital/answerking/exception/util/ValidationErrorResponse.java

```diff
@@ -1,21 +1,19 @@
 package com.answerdigital.answerking.exception.util;
 
 import com.answerdigital.answerking.exception.custom.ValidationException;
-import lombok.Getter;
-import lombok.extern.slf4j.Slf4j;
 
-import javax.servlet.http.HttpServletRequest;
-import java.util.Collection;
-import java.util.Map;
+import jakarta.servlet.http.HttpServletRequest;
+import lombok.SneakyThrows;
+import lombok.extern.slf4j.Slf4j;
 
 @Slf4j
-@Getter
 public class ValidationErrorResponse extends ErrorResponse {
 
-    private final Map<String, Collection<String>> errors;
-
+    @SneakyThrows
```

Comment:
Same as previously mentioned, don't think this is necessary! If it is I'd love to learn about it 😄 

```suggestion

```